### PR TITLE
DFR-2401 config to create a flexible db in preview

### PR DESCRIPTION
**Pull request by gmikerlemos**
https://tools.hmcts.net/jira/browse/DFR-2401

### CODE REVIEW

File: charts/finrem-ccd-definitions/values.preview.template.yaml [hunk 1]
```diff
@@ -208,7 +212,7 @@ ccd:
       imagePullPolicy: Always
       image: hmctspublic.azurecr.io/ccd/definition-store-api:latest
       environment:
-        DEFINITION_STORE_DB_HOST: ${SERVICE_NAME}-postgresql
+        DEFINITION_STORE_DB_HOST: '{{ tpl .Values.global.postgresHostname $}}'
```
Comment (rishikrsharma) [inline charts/finrem-ccd-definitions/values.preview.template.yaml:215]: will this space cause any issue?

**Review by rishikrsharma — APPROVED** [review]

File: charts/finrem-ccd-definitions/values.preview.template.yaml (hunk 1 above)
Comment (gmikerlemos) [reply charts/finrem-ccd-definitions/values.preview.template.yaml:215]: it shouldn't, it is how sscs did on their change

File: charts/finrem-ccd-definitions/values.preview.template.yaml [hunk 2]
```diff
@@ -331,11 +337,11 @@ ccd:
     imageTag: "ccd-divorce-logstash-latest"
     logstashJavaOpts: -Xmx1g -Xms512M
     extraEnvs:
-      - name: DB_URL
-        value: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
-      - name: DB_USER
+      - name: DATA_STORE_PASS
```
Comment (cpareek) [inline charts/finrem-ccd-definitions/values.preview.template.yaml:340]: This need to be set as secret - example - 
https://github.com/hmcts/sscs-tribunals-case-api/blob/e3fdd3fdd32c3e0e99195b4f08f774af1dcf64b6/charts/sscs-tribunals-api/values.ccd.preview.template.yaml#L596-L599

File: charts/finrem-ccd-definitions/values.preview.template.yaml [hunk 3]
```diff
@@ -331,11 +337,11 @@ ccd:
...
         value: hmcts
-      - name: DB_PWD
+      - name: DATA_STORE_URL
+        value: "jdbc:postgresql://sscs-preview.postgres.database.azure.com:5432/{{ .Values.global.databaseNamePrefix }}data-store?sslmode=require&stringtype=unspecified"
```
Comment (cpareek) [inline charts/finrem-ccd-definitions/values.preview.template.yaml:343]: hostname is incorrect

**Pull request closed without merging**